qemu/tests/block_resize.py

```diff
@@ -181,7 +181,7 @@ def get_md5_of_file(filename):
         global current_size
         current_size = 0
         if not wait.wait_for(lambda: verify_disk_size(session, os_type,
-                                                      disk), 20, 0, 1,
+                                                      disk), 20, 0, 3,
```

Comment:
I would like to add a parameter to control it according to OS type if this issue the windows guest just have, since we can save a little time in the linux guest in a way.
Suggestion:
```suggestion
                                                      disk), 20, 0, params.get_numeric("verify_disk_size_steps", 1),
```

cfg:
```
    windows:
        verify_disk_size_steps = 3
```

Reply:
(1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.block_resize.with_iozone.with_system_disk.q35: PASS (586.68 s)

@YongxueHong Updated, thanks~